packages/amplify-e2e-core/src/utils/envVars.ts

```diff
@@ -49,7 +49,7 @@ export function getSocialProviders(getEnv = false): SocialProviders {
     APPLE_APP_ID,
     APPLE_TEAM_ID,
     APPLE_KEY_ID,
-    APPLE_PRIVATE_KEY_2,
+    APPLE_PRIVATE_KEY,
```

Comment:
we were able to clean this up (eliminating the need for _2 private key variables)